css-gaps-1/Overview.bs

```diff
@@ -482,99 +482,217 @@ Pairing gap intersection points into segments</h4>
 		</figure>
 	</div>
 
-<h3 id="outset">
-Adjusting gap decoration endpoints: The 'column-rule-outset', 'row-rule-outset', and 'rule-outset' properties</h3>
+<h3 id="inset">
+Adjusting gap decoration endpoints: The 'rule-inset' properties</h3>
 
 	<pre class='propdef'>
-		Name: column-rule-outset, row-rule-outset
-		Value: <<length-percentage>>
+		Name: column-rule-edge-start-inset, column-rule-edge-end-inset, row-rule-edge-start-inset, row-rule-edge-end-inset
```

Comment:
@SebastianZ had a good point in https://github.com/w3c/csswg-drafts/issues/12603#issuecomment-3458270001 that it would be more consistent to have `start` or `end` as the last word in the property name. Best example is the start/end properties and shorthands in https://www.w3.org/TR/css-logical-1/.